README.md

```diff
@@ -90,8 +81,39 @@ Output file:
 See the [tutorial page](tutorial/readme.md) for sample code,
 as well as the [example gallery](examples/readme.md) to see more of what WireViz can do.
 
+
 ## Usage
 
+### Installation
+
+#### Requirements
+
+WireViz requires Python 3.7 or later.
+
+WireWiz requires GraphViz to be installed in order to work. See the [GraphViz download page](https://graphviz.org/download/) for OS-specific instructions.
+
+_Note_: Ubuntu 18.04 LTS users in particular may need to separately install Python 3.7 or above, as that comes with Python 3.6 as the included system Python install.
+
+#### Installing the latest release
+
+The latest WireViz release can be downloaded from [PyPI](https://pypi.org/project/wireviz/) with the following command:
+```
+pip3 install wireviz
+```
+
+#### Installing the development version
+
+Access to the current state of the development branch can be gained by cloning the repo and installing manually:
+
+```
+git clone <repo url>
+cd <working copy>
+git checkout dev
+pip3 install -e .
```

Comment:
Please make sure this workflow makes sense.

Reply:
Should you also mention something like the alternative variant you gave to me in https://github.com/formatc1702/WireViz/issues/43#issuecomment-651078861?

```sh
git clone https://github.com/formatc1702/WireViz.git
cd WireViz/
python -m venv wv-env
source wv-env/bin/activate
pip3 install -e .
cd src/wireviz
python build_examples.py
```

Note that `git checkout dev` is missing above, and I have not tried any of these installations from scratch recently.

Reply:
Should there be a reference to CONTRIBUTING.md in the section about Installing the development version?

Reply:
Sorry, didn't see these comments earlier.
- I'm not sure whether we should give advice on virtual environments directly, or reference a site that explains the concept more generally.
- Sure, I can include a link to `CONTRIBUTING.md`